### PR TITLE
docs: clarify monitor-grafana-mimir/requirements.md

### PR DESCRIPTION
**Pull request by dimitarvdimitrov**
cAdvisor metrics also need the `instance` label to match what node_exporter and Mimir have

fixes https://github.com/grafana/mimir/issues/7570

### CODE REVIEW

File: docs/sources/mimir/manage/monitor-grafana-mimir/requirements.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ For more information about the kubelet metrics and cAdvisor metrics exported by
 Metrics from kubelet, kube-state-metrics, and cAdvisor must all have a `cluster` label with the same value as in the
 Mimir metrics.
 
-Metrics from node_exporter must all have an `instance` label on them that has the same value as the `instance` label on Mimir metrics.
+Metrics from node_exporter and cAdvisor must all have an `instance` label on them that has the same value as the `instance` label on Mimir metrics.
```
Comment (jdbaldry) [inline docs/sources/mimir/manage/monitor-grafana-mimir/requirements.md:91]: nit: A relatively new addition to the style guide is https://grafana.com/docs/writers-toolkit/write/style-guide/word-list/#node-exporter.

It's up to you if you'd prefer to follow the new style or remain consistent with the one other usage in this file (https://github.com/grafana/mimir/pull/7631/files#diff-48035ca392767c2f32196ce22f0812511181214ec47e9121554718d4307a0fdcR83)

**Review by jdbaldry — APPROVED** [review]

File: docs/sources/mimir/manage/monitor-grafana-mimir/requirements.md (hunk 1 above)
Comment (dimitarvdimitrov) [reply docs/sources/mimir/manage/monitor-grafana-mimir/requirements.md]: i didn't know. thanks for the suggestion. I made the change in https://github.com/grafana/mimir/pull/7631/commits/a12e9aa1ca108cd8b4416d30014f5c2ee9dbb119

Comment (dimitarvdimitrov) [reply docs/sources/mimir/manage/monitor-grafana-mimir/requirements.md]: i'm missing what's the difference between the product and the tool. I decided to go with `node_exporter`. Is this right?

Comment (jdbaldry) [reply docs/sources/mimir/manage/monitor-grafana-mimir/requirements.md]: Ah good question, my interpretation is that it's use "Node exporter" generally and use `node_exporter` if you are talking specifically about the executable

Comment (dimitarvdimitrov) [reply docs/sources/mimir/manage/monitor-grafana-mimir/requirements.md]: aha, in which case we probably want Node Exporter. fixed again in https://github.com/grafana/mimir/pull/7631/commits/38a5faea6d0d30e6c1770ef423f4292563b4b3b1

**Pull request merged**